### PR TITLE
Club: Refining spacing

### PR DESCRIPTION
**Pull request by matiasbenedetto**
#### Changes proposed in this Pull Request:
Club:
- Refining spacing
- Change archives title from H2 to H1 

Index (desktop)

| Before | After |
| --- | --- |
| image | image | 

| Before | After |
| --- | --- |
| image | image |

Index (mobile)

| Before | After |
| --- | --- |
| image | image | 

| Before | After |
| --- | --- |
| image | image | 

Archives (desktop):

| Before | After |
| --- | --- |
| image | image |

Archives (mobile):

| Before | After |
| --- | --- |
| image |image |

### CODE REVIEW

File: club/templates/archive.html [hunk 1]
```diff
@@ -1,6 +1,10 @@
 <!-- wp:template-part {"slug":"header","tagName":"header"} /-->
 
-<!-- wp:query-title {"type":"archive","level":2} /-->
+<!-- wp:query-title {"type":"archive"} /-->
```
Comment (jffng) [inline club/templates/archive.html:3]: I noticed on mobile this increase in font size may easily cut off the word / push beyond the width of the screen:

Screen Shot 2022-08-11 at 3 29 35 PM

File: club/theme.json [hunk 2]
```diff
@@ -366,11 +366,6 @@
                     "style": "solid",
                     "width": "0"
                 },
-                "spacing": {
```
Comment (jffng) [inline club/theme.json:369]: Just want to make sure this is intentional to remove this? I think it looks better but wasn't sure if it's a part of this PR.

**Review by jffng — COMMENTED** [review]
Hello @matiasbenedetto, the PR looks mostly good, I just left two small comments.

File: club/theme.json (hunk 2 above)
Comment (matiasbenedetto) [reply club/theme.json:369]: Yep, it's intentional double spacing at the left of the quote.

File: club/templates/archive.html (hunk 1 above)
Comment (matiasbenedetto) [reply club/templates/archive.html:3]: We are using the font size from the design there.

**Review by jffng — APPROVED** [review]

**Pull request merged**